lib/ldclient-rb/ldclient.rb

```diff
@@ -182,6 +182,31 @@ def track(event_name, user, data)
       add_event({:kind => 'custom', :key => event_name, :user => user, :data => data })
     end
 
+    #
+    # Returns the key of every feature
+    #
+    def feature_keys
+      get_features.map {|feature| feature[:key]}
+    end
+
+    #
+    # Returns all features
+    #
+    def get_features
+      res = @client.get (@config.base_uri + '/api/features') do |req|
```

Comment:
Interesting point. My use case is getting a list of features in order to create methods that we call like:

```ruby
if Labs.feature.some_feature_key? then
...
```

and soon we'll be using it to build a "labs" style opt-in page for users. So I think `/api/features